### PR TITLE
Feed solvent wellrates to output for summary output

### PR DESCRIPTION
**Pull request by totto82**
Depends on OPM/opm-output#99

### CODE REVIEW

**Comment by jokva** [comment]
You really don't need to copy the implementation. It should be sufficient to call the parent report method and then plug in the solvent stuff at the end.

**Comment by totto82** [comment]
Cool. I will try it out.

**Comment by totto82** [comment]
@jokva  Thanks for the tip. It seems to work.

File: opm/autodiff/BlackoilSolventModel_impl.hpp [hunk 1]
```diff
@@ -625,6 +625,19 @@ namespace Opm {
 
         wellModel().updateWellState(dwells, dpMaxRel(), well_state);
 
+        for( auto w = 0; w < wells().number_of_wells; ++w ) {
+            if (wells().type[w] == INJECTOR)
```
Comment (atgeirr) [inline opm/autodiff/BlackoilSolventModel_impl.hpp]: Please add braces here and below.

File: opm/autodiff/BlackoilSolventModel_impl.hpp [hunk 2]
```diff
@@ -625,6 +625,19 @@ namespace Opm {
...
+                continue;
+
+            for (int perf = wells().well_connpos[w]; perf < wells().well_connpos[w+1]; ++perf ) {
+                int wc = wells().well_cells[perf];
+                if ( (ss[wc] + ss[wc]) > 0)
```
Comment (atgeirr) [inline opm/autodiff/BlackoilSolventModel_impl.hpp]: This should probably be ss[wc] + sg[wc]?

File: opm/autodiff/WellStateFullyImplicitBlackoilSolvent.hpp [hunk 3]
```diff
@@ -33,6 +33,37 @@ namespace Opm
...
+            const int np = BaseType::numPhases();
+
+            assert( np == 3 ); // the solvent model assumes 3 phases in the base model
+
+            // completions aren't supported yet
+            for( auto w = 0; w < nw; ++w ) {
+                using rt = data::Rates::opt;
+                double solvent_well_rate = 0.0;
+                for (int perf = wells_->well_connpos[w]; perf < wells_->well_connpos[w+1]; ++perf ) {
+                    auto solvent_rate_this = BaseType::perfPhaseRates()[np*perf + 2] * solventFraction()[perf];
```
Comment (atgeirr) [inline opm/autodiff/WellStateFullyImplicitBlackoilSolvent.hpp]: The `2` here makes me uneasy, I think it should use a phase position and not be hardcoded. In any case magic constants and indices should be named.

**Comment by totto82** [comment]
Ops. I dragged Frankenstein with me. See new PR #848

**Pull request closed without merging**